docs/developer-docs/setup/deploy.mdx

```diff
@@ -0,0 +1,175 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Deploy
+
+## Overview
+
+After you have built a canister, you can deploy it either on the local execution environment or on the mainnet. Your canister must be deployed before you can make calls to it and execute methods. 
+
+- #### Step 1:  Open a new terminal and navigate to your project directory.
+
+- #### Step 2:  Start the local canister execution environment, if necessary.
+
+```
+dfx start
+```
+
+- #### Step 3:  Verify the canisters you'd like to deploy are configured in the project's `dfx.json` file.
+
+An example `dfx.json` file can be found in the [default project template document](/docs/current/developer-docs/setup/first-canister)
+
+- #### Step 4:  Deploy all of the canisters within your project by running the following command:
```

Comment:
I would suggest either having 1) one step for deployment and a section for local or mainnet deployment each OR 2) separate steps for local and mainnet deployment like you have here but specify that Step 4 is for local deployment.

Reply:
Included steps for local/mainnet/playground deployments